src/geo/map/cartodb-layer.js

```diff
@@ -5,9 +5,16 @@ var InfowindowTemplate = require('./infowindow-template');
 var TooltipTemplate = require('./tooltip-template');
 var Legends = require('./legends/legends');
 var AnalysisModel = require('../../analysis/analysis-model');
+var Aggregation = require('../../api/v4/layer/aggregation');
```

Comment:
Not sure about requiring objects of the public API (`api/v4`) from the internal library code.

In this case it's required only to get the placement constant but it's not needed to trigger a default aggregation.